setup.cfg

```diff
@@ -34,7 +34,7 @@ install_requires =
     more-itertools
     dataclasses; python_version=="3.6"
 tests_require =
-    pytest>=5.0.0
+    pytest>=6.0.0
```

Comment:
Since we are always using pytest to run the tests, `tests_require` no longer used anywhere and can be removed.

Reply:
Thanks.